src/osemgrep/language_server/server/LS_metrics.ml

```diff
@@ -0,0 +1,49 @@
+(* Austin Theriault
+ *
+ * Copyright (C) 2019-2023 Semgrep, Inc.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public License
+ * version 2.1 as published by the Free Software Foundation, with the
+ * special exception on linking described in file LICENSE.
+ *
+ * This library is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the file
+ * LICENSE for more details.
+ *)
+
+(* Commentary *)
+(* Metrics we send to the mothership *)
```

Comment:
maybe a small comment about how it differs from src/osemgrep/core/Metrics_.ml, and semgrep-interfaces/semgrep_metrics.atd ?